docs/source/docs/examples/aimandrange.md

```diff
@@ -4,36 +4,47 @@ The following example is from the PhotonLib example repository ([Java](https://g
 
 ## Knowledge and Equipment Needed
 
-- Everything required in {ref}`Aiming at a Target <docs/examples/aimingatatarget:Knowledge and Equipment Needed>` and {ref}`Getting in Range of the Target <docs/examples/gettinginrangeofthetarget:Knowledge and Equipment Needed>`.
+- Everything required in {ref}`Aiming at a Target <docs/examples/aimingatatarget:Knowledge and Equipment Needed>`.
 
 ## Code
 
-Now that you know how to both aim and get in range of the target, it is time to combine them both at the same time. This example will take the previous two code examples and make them into one function using the same tools as before. With this example, you now have all the knowledge you need to use PhotonVision on your robot in any game.
+Now that you know how to aim toward the AprilTag, let's also drive the correct distance from the AprilTag.
+
+To do this, we'll use the *pitch* of the target in the camera image and trigonometry to figure out how far away the robot is from the AprilTag. Then, like before, we'll use the P term of a PID controller to drive the robot to the correct distance.
 
 ```{eval-rst}
 .. tab-set::
 
     .. tab-item:: Java
 
-       .. rli:: https://raw.githubusercontent.com/PhotonVision/photonvision/ebef19af3d926cf87292177c9a16d01b71219306/photonlib-java-examples/aimandrange/src/main/java/frc/robot/Robot.java
+       .. rli:: https://raw.githubusercontent.com/gerth2/photonvision/adb3098fbe0cdbc1a378c6d5a41126dd1d6d6955/photonlib-java-examples/aimandrange/src/main/java/frc/robot/Robot.java
          :language: java
-         :lines: 42-111
+         :lines: 84-131
          :linenos:
-         :lineno-start: 42
+         :lineno-start: 84
 
     .. tab-item:: C++ (Header)
 
-       .. rli:: https://raw.githubusercontent.com/PhotonVision/photonvision/ebef19af3d926cf87292177c9a16d01b71219306/photonlib-cpp-examples/aimandrange/src/main/include/Robot.h
-         :language: cpp
-         :lines: 27-71
+       .. rli:: https://raw.githubusercontent.com/gerth2/photonvision/adb3098fbe0cdbc1a378c6d5a41126dd1d6d6955/photonlib-cpp-examples/aimandrange/src/main/include/Robot.h
```

Comment:
RLI pointing to a fork? Might need to be updated separately, not sure

Reply:
Yea I tossed in an issue but this needs to be a two-step, we won't know the git sha1 of the master merge until we merge it.... using my fork for now.

Reply:
https://github.com/PhotonVision/photonvision/issues/1418 future-us problem